docs/monitoring/docs/030-gs-monitoring-custom-metrics.md

```diff
@@ -0,0 +1,145 @@
+
+    kubectl apply -f https://raw.githubusercontent.com/kyma-project/examples/master/monitoring-custom-metrics/deployment/service-monitor.yaml
+    ```
+
+    ```bash
+    kubectl get pods
+    NAME                             READY     STATUS    RESTARTS   AGE
+    sample-metrics-c9f998959-jd2fz   2/2       Running   0          2m
+    sample-metrics-c9f998959-kfbp8   2/2       Running   0          2m
+    sample-metrics-c9f998959-nnp2n   2/2       Running   0          2m
+    sample-metrics-c9f998959-vdnkn   2/2       Running   0          2m
+    ```
+
+2. Run the `port-forward` command on the `sample-metrics-8081` service for the`8081` port to check the metrics.
+    ```bash
+    kubectl port-forward svc/sample-metrics-8081 8081:8081
+    ```
+    Then open a browser and access [`http://localhost:8081/metrics`](http://localhost:8081/metrics)
+    
+    ![metrics on port 8081](assets/sample-metrics-2.png)
+
+Find the source code for the sample application [here](https://github.com/kyma-project/examples/blob/master/monitoring-custom-metrics/main.go). See the [package prometheus](https://godoc.org/github.com/prometheus/client_golang/prometheus) for the reference documentation. Read [this](https://prometheus.io/docs/concepts/metric_types/) documentation to learn more about the Prometheus metric types.
+
+### Access the exposed metrics in Prometheus
+
+ Run the `port-forward` command on the `core-prometheus` service:
+    
+```bash
+kubectl port-forward svc/core-prometheus -n kyma-system 9090:9090
+Forwarding from 127.0.0.1:9090 -> 9090
+Forwarding from [::1]:9090 -> 9090
+```
+All the **sample-metrics** endpoints appear as the [`Targets`](http://localhost:9090/targets#job-sample-metrics-8081) list.
+
+![Prometheus Dashboard](assets/pm-dashboard-1.png)
+
+Use either the `cpu_temperature_celsius` or `hd_errors_total` in the [**expression**](http://localhost:9090/graph) field. Click the **Execute** button to check the values scrapped by Prometheus.
+
+![Prometheus Dashboard](assets/pm-dashboard-2.png)
+
+### ServiceMonitor
+
+Prometheus can reach the service using ServiceMonitor. ServiceMonitor is a specific CRD used by the Prometheus operator to monitor services.
+
+In Kyma, the Prometheus server discovers all ServiceMonitors through the **serviceMonitorSelector** matching the `prometheus: core` label.
+
+```yaml
+  serviceMonitorSelector:
+    matchLabels:
+      prometheus: {{ .Values.prometheusLabelValue | default .Release.Name | quote }}
+{{- end }}
+```
+
+In this example, the ServiceMonitor selects a **selector** with all services matching the `k8s-app: metrics` label. Find the complete yaml [here](https://github.com/kyma-project/examples/blob/master/monitoring-custom-metrics/deployment/service-monitor.yaml).
+
+In Kyma, there is a [template](https://github.com/kyma-project/kyma/blob/master/resources/core/charts/monitoring/charts/prometheus/templates/servicemonitors.yaml), which serves to discover a list of ServiceMonitors.
```

Comment:
No comma after the link (sorry, my mistake)

Reply:
done